src/app/onion/old-dashboard/components/side-panel-content/learning-object/learning-object.component.scss

```
@@ -1,5 +1,11 @@
 @import '~_vars.scss';
 
+.learning-object {
+  padding: 15px;
+  background: white;
+  border-radius: 4px;
+  box-shadow: 0 0 10px 1px $medium-grey;
```

Can we use an rgba variant of black here? Off the cuff suggestion so play around with it, but using the medium grey shadow on the blue makes the shadow look white at the top.
```suggestion
  box-shadow: 0 0 10px 1px rgba(0, 0, 0, 0.07);
```